javaslang/src/main/java/javaslang/Value.java

```diff
@@ -96,6 +97,7 @@
  * <li>{@link #toMap(Function)}</li>
  * <li>{@link #toOption()}</li>
  * <li>{@link #toQueue()}</li>
+ * <li>{@link #toSortedQueue(Comparator)}</li>
```

Comment:
Good! Most committers forget to update this section :) Please sort it alphabetically like the other items.